### PR TITLE
Task

### PR DESCRIPTION
**Pull request by HasaluSL**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a live link or preview screenshot
- [ ] Images are `240 x 240` [w x h]
- [ ] Resolved merge conflicts
- [ ] Included a description of my change below

# Things done in this Pull Request

- Replace this with your change description

### CODE REVIEW

File: Interview a Student participating in the Contest [hunk 1]
```diff
@@ -0,0 +1 @@
+https://introductorymusicblockstutorial.wordpress.com/
```
Comment (abishekvashok) [inline Interview a Student participating in the Contest:1]: Pls don't add this.

File: index.html [hunk 2]
```diff
@@ -146,12 +146,12 @@ <h2 class="animated fadeIn">Students just like you.</h2>+
 						</a>
 						{% endif %}
 						{% if student.twitter %}
-						<a href="https://twitter.com/@{{student.twitter}}" class="icon-a">
+						<a href="https://twitter.com/@{{cyberdoc.twitter}}" class="icon-a">
```
Comment (abishekvashok) [inline index.html:149]: Add yourself to _data/students.yml and not here. Pls see the readme.md

File: index.html [hunk 3]
```diff
@@ -772,6 +772,15 @@ <h4 class="uppercase mb16">Lightbox Grid Gallery</h4>
 								<div class="background-image-holder">
 									<img alt="image" class="background-image" src="img/cover8.jpg">
 								</div>
+								<div class="row">
+					<div class="col-md-3 col-sm-6">
+						<a href=https://introductorymusicblockstutorial.wordpress.com/ target="_blank">
+							<div class="image-tile outer-title text-center">
+								<img alt="Pic" src="img/blog/1.jpeg" height="240" width="240">
+								<div class="title mb16">
```
Comment (abishekvashok) [inline index.html:780]: Add this to blogs.yml see the readme

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Comment by anshumanv** [comment]
Duplicate of #258

**Review by isuruAb — COMMENTED** [review]
@HasaluSL  please describe what you have changed and why. cheers 👍

**Comment by ParthS007** [comment]
Closing this as a duplicate of #259

**Pull request closed without merging**